src/search_datastructures.hpp

```diff
@@ -423,6 +438,7 @@ class GlobalDataHolder
     TBamFile            outfileBam;
 
     TScoreScheme3       scoringScheme;
```

Comment:
```suggestion
    TScoreScheme3       scoringSchemePreScoring;
```
Maybe denote clearly what this is used for?